### ISSUE TITLE
There is no back / forward buttons

### ISSUE THREAD

**Issue by damusix** [brave/browser-android-tabs#56]
Not sure if maybe I'm not seeing them. This makes it difficult to power-surf.

**Comment by HennyBro** [comment]
You are right. But before today there weren't forward or back GESTURES in Chromium as well.

**Comment by HennyBro** [comment]
I suggest you to close this thread, because I'm going to push another one about gesture back and forward navigation in Brave for Android/iOS.
